src/main/java/org/opentripplanner/framework/time/TimeUtils.java

```diff
@@ -246,4 +248,8 @@ public static long busyWait(int waitMs) {
     }
     return value;
   }
+
+  public static int toTransitTimeSeconds(ZonedDateTime transitSearchTimeZero, Instant time) {
```

Comment:
Can you document what kind of values this returns? Can it be negative if `transitSearchTime` is before `time`?